tests/manual/text/INVISIBLE_GLYPH_IDTest.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (c) 2018, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Can you update this to 2019?

Reply:
I have no time today, so I will do it tomorrow.

Reply:
I did re-push to https://github.com/javafxports/openjdk-jfx/commit/d2edbed328a753f536bcb4c92cae874b2097646f.

Only INVISIBLE_GLYPH_IDTest.java is changed.
Copyright 2018 -> 2019
and added replace("\\", "/") to run collectry on Windows10 1809.
`userHome = userHome.replace("\\", "/");`